docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
```

Comment:
What is the main purpose and who are the target readers of this section? It looks like design and internal implementation? If so, I would suggest to move it to the last sub-section and name it `IPAM design`.

Reply:
This is to explain how antrea manages IP when FlexibleIPAM feature is enabled.
Moved it to the last sub-section and name it `Flexible IPAM design`.